cogs/quote_commands.py

```diff
@@ -21,19 +21,8 @@ class QuoteCommands(utils.Cog):
 
     IMAGE_URL_REGEX = re.compile(r"(http(?:s?):)([/|.|\w|\s|-])*\.(?:jpg|gif|png|jpeg|webp)")
     QUOTE_SEARCH_CHARACTER_CUTOFF = 100
-
-    @utils.group(invoke_without_command=True)
-    @commands.bot_has_permissions(send_messages=True, embed_links=True, add_reactions=True)
-    @commands.guild_only()
-    async def quote(self, ctx:utils.Context, messages:commands.Greedy[discord.Message]):
-        """
-        Quotes a user's message to the guild's quote channel.
-        """
-
-        # Make sure no subcommand is passed
-        if ctx.invoked_subcommand is not None:
-            return
-
+    
+    async def startquote(self, ctx, messages):
```

Comment:
Change the name of this method - `get_quote_messages` would be more apt.

This method as a whole also needs to be changed entirely anyway - sometimes you're returning a message, sometimes an embed. Be consistent. You can't just copy/paste my code into a new method and hope it works.